content/de/act/honey/index.md

```diff
@@ -224,3 +229,31 @@ window.addEventListener('load', function() {
     });
 });
 </script>
+
+
+## Wie kann ich mich darüber beschweren?
+
+Die DSGVO gibt Dir das Recht Dich bei den unabhängigen {{< link slug="supervisory-authorities" text="Datenschutz-Aufsichtsbehörden" >}} über Unternehmen zu beschweren. Du kannst dieses Tool nutzen, um die für Dich zuständige Behörde und ihre Kontaktdetails zu finden: 
```

Comment:
Since we expect this to be read by quite a large audience, this needs to have more context. I'd mention at least:

* after access request
* if you feel Honey violated your rights based on the data they gave you
* what can/will the DPA do with the complaint?
* complaint is free
* maybe: expect this to take a while